### ISSUE TITLE
Strange TemplateNotFound Error

### ISSUE THREAD

**Issue by LloydArmstrong** [saltstack-formulas/logstash-formula#32]
Hi All,

I have the formula cloned as it comes, where I have specified the config in a pillar file, with no filters, and output pointing to elasticsearch as shown:
```
logstash:
  java: openjdk-8-jre-headless
  use_upstream_repo: True
  repo:
      version: 6
      old_repo: False
  inputs:
    -
      plugin_name: file
      path:
        - /var/log/syslog
        - /var/log/auth.log
      type: syslog
  outputs:
    -
      plugin_name: elasticsearch
      hosts:
        - "elastic7.myserver.com"
      port: 9200
```
Nothing else has been changed from the cloned repo. However, I get the following error upon applying the state:
```
          ID: logstash-config-inputs
    Function: file.managed
        Name: /etc/logstash/conf.d/01-inputs.conf
      Result: False
     Comment: Unable to manage file: Jinja error: logstash/map.jinja
              Traceback (most recent call last):
                File "/var/tmp/.salt_67a7f8_salt/py2/salt/utils/templates.py", line 389, in render_jinja_tmpl
                  output = template.render(**decoded_context)
                File "/var/tmp/.salt_67a7f8_salt/py2/jinja2/environment.py", line 1008, in render
                  return self.environment.handle_exception(exc_info, True)
                File "/var/tmp/.salt_67a7f8_salt/py2/jinja2/environment.py", line 780, in handle_exception
                  reraise(exc_type, exc_value, tb)
                File "<template>", line 1, in top-level template code
                File "/var/tmp/.salt_67a7f8_salt/py2/salt/utils/jinja.py", line 140, in get_source
                  raise TemplateNotFound(template)
              TemplateNotFound: logstash/map.jinja

              ; line 1

              ---
              {%- from 'logstash/map.jinja' import logstash, format_logstash_config with context -%}    <======================
              {{ format_logstash_config('input', logstash.inputs) }}
              Traceback (most recent call last):
                File "/var/tmp/.salt_67a7f8_salt/py2/salt/utils/templates.py", line 389, in render_jinja_tmpl
                  output = template.render(**decoded_context)
                File "/var/tmp/.salt_67a7f8_salt/py2/jinja2/environment.py", line 1008, in render
              [...]
              ---
     Started: 11:56:16.988834
    Duration: 45.645 ms
     Changes:
```
I'm unsure what the issue may be because as far as I can see on my side, everything is where it should be. Perhaps the formula does not support ES? Please let me know if this is a legitimate bug, or if there is something outstanding in my configuration.

**Many Thanks!**

**Comment by LloydArmstrong** [comment]
Bump. Anyone still using this formula?

**Comment by myoung34** [comment]
My version from many months still works.
Yours says it cant find the template but [its definitely here](https://github.com/saltstack-formulas/logstash-formula/blob/master/logstash/map.jinja) are you sure its being copied/compiled and actually exists?

**Comment by LloydArmstrong** [comment]
Thanks @myoung34 for the feedback. I was under the impression that everything was in place and should be working. I have recently taken over a relatively large stack with more than a dozen formulae in use. We use Salt-SSH to test our container builds and I found that I had not referenced `salt://logstash/map.jinja` in the `salt-ssh` section of our Saltfile. Once I did that, it all worked.

You Live and Learn!
